pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>me.sfiguz7</groupId>
   <artifactId>ExtraTools</artifactId>
-  <version>1.2</version>
+  <version>MODIFIED</version>
```

Comment:
>:(

Reply:
I mean... You use busy build page so might aswell >:)